documentation/content/en/books/handbook/cutting-edge/_index.adoc

```diff
@@ -1137,7 +1137,7 @@ set `PACKAGES` on the build system to a directory similar to `DISTDIR`.
 == Building on non-FreeBSD Hosts
 
 Historically, building FreeBSD required a FreeBSD host.
-Nowadays, the FreeBSD can be build on Linux distributions and macOS.
+Nowadays, FreeBSD can be built on Linux distributions and macOS.
```

Comment:
```suggestion
Nowadays, FreeBSD can be built on Linux and macOS.
```

Technical neat to say _Linux distributions_, however yer average reader will understand what's meant by _Linux_.